app/src/main/java/org/sopt/sample/data/repository/HomeRepository.kt

```diff
@@ -0,0 +1,11 @@
+package org.sopt.sample.data.repository
+
+import org.sopt.sample.data.model.GithubInformation
+import org.sopt.sample.data.source.HomeDataSourceImpl
+
+class HomeRepository(private val homeDataSourceImpl: HomeDataSourceImpl) {
+
+    fun getGithubInformations(): List<GithubInformation>? {
+        return homeDataSourceImpl.getGithubInformations()
+    }
```

Comment:
```suggestion
    fun getGithubInformations(): List<GithubInformation>? = homeDataSourceImpl.getGithubInformations()
```
꼭 바꾸지 않아도 괜찮지만 이렇게 표현할 수 있다는 거 다 압니다 :)